packages/frontend-2/components/project/webhooks-page/CreateWebhookDialog.vue

```diff
@@ -0,0 +1,140 @@
+<template>
```

Comment:
I was able to create a webhook without any events selected. Not only that, I was able to create a webhook without any fields filled out at all! Please add validation to all forms to prevent this from happening.

Reply:
Working on triggers validation now

Reply:
Should be working now on origin